


 时间 2018-05-30 15:29:50.8720 
 消息内容 requestLog
{"URI":"/api/Values","ElapsedTime":46.465900000000005,"Method":"GET","RequestParams":"","QueryString":"","Result":"[\"value1\",\"value2\"]","Status":200,"Headers":{"Connection":["keep-alive"],"Accept":["application/json"],"Accept-Encoding":["gzip, deflate, br"],"Accept-Language":["zh-CN,zh;q=0.9"],"Host":["localhost:29162"],"Referer":["http://localhost:29162/swagger/ui/index"],"User-Agent":["Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.181 Safari/537.36"]},"ChildrenRequests":{},"DBLogs":[],"Exception":null}
 线程ID 11 



 时间 2018-05-30 15:30:12.9932 
 消息内容 requestLog
{"URI":"/api/Values/000","ElapsedTime":25.7392,"Method":"DELETE","RequestParams":"","QueryString":"","Result":"","Status":204,"Headers":{"Connection":["keep-alive"],"Accept":["*/*"],"Accept-Encoding":["gzip, deflate, br"],"Accept-Language":["zh-CN,zh;q=0.9"],"Host":["localhost:29162"],"Referer":["http://localhost:29162/swagger/ui/index"],"User-Agent":["Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.181 Safari/537.36"],"Origin":["http://localhost:29162"]},"ChildrenRequests":{},"DBLogs":[],"Exception":null}
 线程ID 16 



 时间 2018-05-30 15:30:38.3157 
 消息内容 requestLog
{"URI":"/api/Values","ElapsedTime":12.71,"Method":"POST","RequestParams":"999","QueryString":"","Result":"","Status":204,"Headers":{"Connection":["keep-alive"],"Content-Length":["3"],"Content-Type":["application/json"],"Accept":["*/*"],"Accept-Encoding":["gzip, deflate, br"],"Accept-Language":["zh-CN,zh;q=0.9"],"Host":["localhost:29162"],"Referer":["http://localhost:29162/swagger/ui/index"],"User-Agent":["Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.181 Safari/537.36"],"Origin":["http://localhost:29162"]},"ChildrenRequests":{},"DBLogs":[],"Exception":null}
 线程ID 29 



 时间 2018-05-30 15:32:42.8056 
 消息内容 requestLog
{"URI":"/api/Values/000","ElapsedTime":160.6844,"Method":"DELETE","RequestParams":"","QueryString":"","Result":"{\"Code\":500,\"SubCode\":\"unknow-error\",\"Msg\":\"系统繁忙！\"}","Status":200,"Headers":{"Connection":["keep-alive"],"Accept":["*/*"],"Accept-Encoding":["gzip, deflate, br"],"Accept-Language":["zh-CN,zh;q=0.9"],"Host":["localhost:29162"],"Referer":["http://localhost:29162/swagger/ui/index"],"User-Agent":["Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.181 Safari/537.36"],"Origin":["http://localhost:29162"]},"ChildrenRequests":{},"DBLogs":[],"Exception":"Message:xxx\r\nStackTrace   在 WebTest2.Controllers.ValuesController.Delete(Int32 id) 位置 D:\\源码\\Allen\\MF\\Tests\\WebTest2\\Controllers\\ValuesController.cs:行号 39\r\n   在 lambda_method(Closure , Object , Object[] )\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass6_0.<GetExecutor>b__0(Object instance, Object[] methodParameters)\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.Execute(Object instance, Object[] arguments)\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__1.MoveNext()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__5.MoveNext()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__6.MoveNext()"}
 线程ID 39 



 时间 2018-05-30 15:32:55.3965 
 消息内容 requestLog
{"URI":"/api/Values/2","ElapsedTime":17.5169,"Method":"PUT","RequestParams":"fff","QueryString":"","Result":"{\"Code\":400,\"SubCode\":\"\",\"Msg\":\"xxx\"}","Status":200,"Headers":{"Connection":["keep-alive"],"Content-Length":["3"],"Content-Type":["application/json"],"Accept":["*/*"],"Accept-Encoding":["gzip, deflate, br"],"Accept-Language":["zh-CN,zh;q=0.9"],"Host":["localhost:29162"],"Referer":["http://localhost:29162/swagger/ui/index"],"User-Agent":["Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.181 Safari/537.36"],"Origin":["http://localhost:29162"]},"ChildrenRequests":{},"DBLogs":[],"Exception":"Message:xxx\r\nStackTrace   在 WebTest2.Controllers.ValuesController.Put(Int32 id, String value) 位置 D:\\源码\\Allen\\MF\\Tests\\WebTest2\\Controllers\\ValuesController.cs:行号 33\r\n   在 lambda_method(Closure , Object , Object[] )\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass6_0.<GetExecutor>b__0(Object instance, Object[] methodParameters)\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.Execute(Object instance, Object[] arguments)\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__1.MoveNext()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__5.MoveNext()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__6.MoveNext()"}
 线程ID 13 



 时间 2018-05-30 15:41:35.4936 
 消息内容 requestLog
{"URI":"/api/Values","ElapsedTime":119.09060000000001,"Method":"GET","RequestParams":"","QueryString":"","Result":"[\"value1\",\"value2\"]","Status":200,"Headers":{"Connection":["keep-alive"],"Accept":["application/json"],"Accept-Encoding":["gzip, deflate, br"],"Accept-Language":["zh-CN,zh;q=0.9"],"Host":["localhost:29162"],"Referer":["http://localhost:29162/swagger/ui/index"],"User-Agent":["Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.181 Safari/537.36"]},"ChildrenRequests":{},"DBLogs":[],"Exception":null}
 线程ID 27 



 时间 2018-05-30 15:41:52.5347 
 消息内容 requestLog
{"URI":"/api/Values/222","ElapsedTime":35.838300000000004,"Method":"GET","RequestParams":"","QueryString":"","Result":"{\"Msg\":\"Success\",\"Code\":200,\"Data\":{\"Id\":\"c28c13f6-7cc5-477e-8146-6adeacfc9303\"}}","Status":200,"Headers":{"Connection":["keep-alive"],"Accept":["application/json"],"Accept-Encoding":["gzip, deflate, br"],"Accept-Language":["zh-CN,zh;q=0.9"],"Host":["localhost:29162"],"Referer":["http://localhost:29162/swagger/ui/index"],"User-Agent":["Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.181 Safari/537.36"]},"ChildrenRequests":{},"DBLogs":[],"Exception":null}
 线程ID 5 



 时间 2018-05-30 15:42:04.4989 
 消息内容 requestLog
{"URI":"/api/Values","ElapsedTime":15.066400000000002,"Method":"POST","RequestParams":"999","QueryString":"","Result":"{\"Msg\":\"Success\",\"Code\":200,\"Data\":[\"1111\",\"2222\"]}","Status":200,"Headers":{"Connection":["keep-alive"],"Content-Length":["3"],"Content-Type":["application/json"],"Accept":["*/*"],"Accept-Encoding":["gzip, deflate, br"],"Accept-Language":["zh-CN,zh;q=0.9"],"Host":["localhost:29162"],"Referer":["http://localhost:29162/swagger/ui/index"],"User-Agent":["Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.181 Safari/537.36"],"Origin":["http://localhost:29162"]},"ChildrenRequests":{},"DBLogs":[],"Exception":null}
 线程ID 8 



 时间 2018-05-30 15:43:10.9494 
 消息内容 requestLog
{"URI":"/api/Values","ElapsedTime":139.74790000000002,"Method":"POST","RequestParams":"999","QueryString":"","Result":"{\"Code\":500,\"SubCode\":\"unknow-error\",\"Msg\":\"系统繁忙！\"}","Status":200,"Headers":{"Connection":["keep-alive"],"Content-Length":["3"],"Content-Type":["application/json"],"Accept":["*/*"],"Accept-Encoding":["gzip, deflate, br"],"Accept-Language":["zh-CN,zh;q=0.9"],"Host":["localhost:29162"],"Referer":["http://localhost:29162/swagger/ui/index"],"User-Agent":["Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.181 Safari/537.36"],"Origin":["http://localhost:29162"]},"ChildrenRequests":{},"DBLogs":[],"Exception":"Message:未将对象引用设置到对象的实例。\r\nStackTrace   在 MF.Core.ApiResponsePager`1..ctor(IPagedList`1 pagerData)\r\n   在 WebTest2.Controllers.ValuesController.Post(String value) 位置 D:\\源码\\Allen\\MF\\Tests\\WebTest2\\Controllers\\ValuesController.cs:行号 38\r\n   在 lambda_method(Closure , Object , Object[] )\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass6_1.<GetExecutor>b__3(Object instance, Object[] methodParameters)\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.Execute(Object instance, Object[] arguments)\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__1.MoveNext()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__5.MoveNext()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__6.MoveNext()"}
 线程ID 10 



 时间 2018-05-30 15:43:42.4243 
 消息内容 requestLog
{"URI":"/api/Values/000","ElapsedTime":179.07930000000002,"Method":"DELETE","RequestParams":"","QueryString":"","Result":"{\"Code\":500,\"SubCode\":\"unknow-error\",\"Msg\":\"系统繁忙！\"}","Status":200,"Headers":{"Connection":["keep-alive"],"Accept":["*/*"],"Accept-Encoding":["gzip, deflate, br"],"Accept-Language":["zh-CN,zh;q=0.9"],"Host":["localhost:29162"],"Referer":["http://localhost:29162/swagger/ui/index"],"User-Agent":["Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.181 Safari/537.36"],"Origin":["http://localhost:29162"]},"ChildrenRequests":{},"DBLogs":[],"Exception":"Message:xxx\r\nStackTrace   在 WebTest2.Controllers.ValuesController.Delete(Int32 id) 位置 D:\\源码\\Allen\\MF\\Tests\\WebTest2\\Controllers\\ValuesController.cs:行号 51\r\n   在 lambda_method(Closure , Object , Object[] )\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass6_0.<GetExecutor>b__0(Object instance, Object[] methodParameters)\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.Execute(Object instance, Object[] arguments)\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__1.MoveNext()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__5.MoveNext()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__6.MoveNext()"}
 线程ID 23 



 时间 2018-05-30 15:44:09.3263 
 消息内容 requestLog
{"URI":"/api/Values/222","ElapsedTime":134.9539,"Method":"GET","RequestParams":"","QueryString":"","Result":"{\"Msg\":\"Success\",\"Code\":200,\"Data\":{\"Id\":\"4d8889ef-cfa0-40ed-a5ad-0bc4f9caa5aa\"}}","Status":200,"Headers":{"Connection":["keep-alive"],"Accept":["application/json"],"Accept-Encoding":["gzip, deflate, br"],"Accept-Language":["zh-CN,zh;q=0.9"],"Host":["localhost:29162"],"Referer":["http://localhost:29162/swagger/ui/index"],"User-Agent":["Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.181 Safari/537.36"]},"ChildrenRequests":{},"DBLogs":[],"Exception":null}
 线程ID 24 



 时间 2018-05-30 15:44:09.7454 
 消息内容 requestLog
{"URI":"/api/Values/000","ElapsedTime":14.1812,"Method":"DELETE","RequestParams":"","QueryString":"","Result":"{\"Code\":500,\"SubCode\":\"unknow-error\",\"Msg\":\"系统繁忙！\"}","Status":200,"Headers":{"Connection":["keep-alive"],"Accept":["*/*"],"Accept-Encoding":["gzip, deflate, br"],"Accept-Language":["zh-CN,zh;q=0.9"],"Host":["localhost:29162"],"Referer":["http://localhost:29162/swagger/ui/index"],"User-Agent":["Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.181 Safari/537.36"],"Origin":["http://localhost:29162"]},"ChildrenRequests":{},"DBLogs":[],"Exception":"Message:xxx\r\nStackTrace   在 WebTest2.Controllers.ValuesController.Delete(Int32 id) 位置 D:\\源码\\Allen\\MF\\Tests\\WebTest2\\Controllers\\ValuesController.cs:行号 51\r\n   在 lambda_method(Closure , Object , Object[] )\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass6_0.<GetExecutor>b__0(Object instance, Object[] methodParameters)\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.Execute(Object instance, Object[] arguments)\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__1.MoveNext()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__5.MoveNext()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__6.MoveNext()"}
 线程ID 9 



 时间 2018-05-30 15:44:36.2549 
 消息内容 requestLog
{"URI":"/api/Values/000","ElapsedTime":164.5187,"Method":"DELETE","RequestParams":"","QueryString":"","Result":"{\"Code\":500,\"SubCode\":\"unknow-error\",\"Msg\":\"系统繁忙！\"}","Status":200,"Headers":{"Connection":["keep-alive"],"Accept":["*/*"],"Accept-Encoding":["gzip, deflate, br"],"Accept-Language":["zh-CN,zh;q=0.9"],"Host":["localhost:29162"],"Referer":["http://localhost:29162/swagger/ui/index"],"User-Agent":["Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.181 Safari/537.36"],"Origin":["http://localhost:29162"]},"ChildrenRequests":{},"DBLogs":[],"Exception":"Message:xxx\r\nStackTrace   在 WebTest2.Controllers.ValuesController.Delete(Int32 id) 位置 D:\\源码\\Allen\\MF\\Tests\\WebTest2\\Controllers\\ValuesController.cs:行号 51\r\n   在 lambda_method(Closure , Object , Object[] )\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass6_0.<GetExecutor>b__0(Object instance, Object[] methodParameters)\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.Execute(Object instance, Object[] arguments)\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__1.MoveNext()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__5.MoveNext()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__6.MoveNext()"}
 线程ID 24 



 时间 2018-05-30 15:49:29.1843 
 消息内容 requestLog
{"URI":"/api/Values/000","ElapsedTime":4878.0532,"Method":"DELETE","RequestParams":"","QueryString":"","Result":"{\"Code\":500,\"SubCode\":\"unknow-error\",\"Msg\":\"系统繁忙！\"}","Status":200,"Headers":{"Connection":["keep-alive"],"Accept":["*/*"],"Accept-Encoding":["gzip, deflate, br"],"Accept-Language":["zh-CN,zh;q=0.9"],"Host":["localhost:29162"],"Referer":["http://localhost:29162/swagger/ui/index"],"User-Agent":["Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.181 Safari/537.36"],"Origin":["http://localhost:29162"]},"ChildrenRequests":{},"DBLogs":[],"Exception":"Message:xxx\r\nStackTrace   在 WebTest2.Controllers.ValuesController.Delete(Int32 id) 位置 D:\\源码\\Allen\\MF\\Tests\\WebTest2\\Controllers\\ValuesController.cs:行号 51\r\n   在 lambda_method(Closure , Object , Object[] )\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass6_0.<GetExecutor>b__0(Object instance, Object[] methodParameters)\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.Execute(Object instance, Object[] arguments)\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__1.MoveNext()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__5.MoveNext()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__6.MoveNext()"}
 线程ID 6 



 时间 2018-05-30 15:59:45.9203 
 消息内容 requestLog
{"URI":"/api/Values","ElapsedTime":8998.4984,"Method":"POST","RequestParams":"7777","QueryString":"","Result":"{\"Code\":500,\"SubCode\":\"unknow-error\",\"Msg\":\"系统繁忙！\"}","Status":200,"Headers":{"Connection":["keep-alive"],"Content-Length":["4"],"Content-Type":["application/json"],"Accept":["application/json"],"Accept-Encoding":["gzip, deflate, br"],"Accept-Language":["zh-CN,zh;q=0.9"],"Host":["localhost:29162"],"Referer":["http://localhost:29162/swagger/ui/index"],"User-Agent":["Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.181 Safari/537.36"],"Origin":["http://localhost:29162"]},"ChildrenRequests":{},"DBLogs":[],"Exception":"Message:未将对象引用设置到对象的实例。\r\nStackTrace   在 MF.Core.ApiResponsePager`1..ctor(IPagedList`1 pagerData)\r\n   在 WebTest2.Controllers.ValuesController.Post(String value) 位置 D:\\源码\\Allen\\MF\\Tests\\WebTest2\\Controllers\\ValuesController.cs:行号 38\r\n   在 lambda_method(Closure , Object , Object[] )\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass6_1.<GetExecutor>b__3(Object instance, Object[] methodParameters)\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.Execute(Object instance, Object[] arguments)\r\n   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__1.MoveNext()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__5.MoveNext()\r\n--- 引发异常的上一位置中堆栈跟踪的末尾 ---\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__6.MoveNext()"}
 线程ID 5 
